packages/orbit-components/src/NotificationBadge/NotificationBadge.stories.tsx

```diff
@@ -8,236 +9,86 @@ import NotificationBadge from ".";
 
 const getIcon = (source: string | null) => source && Icons[source];
 
-export default {
+const meta: Meta<typeof NotificationBadge> = {
   title: "NotificationBadge",
-};
-
-export const Default = ({ content, icon }) => {
-  const Icon = getIcon(icon);
-  return <NotificationBadge icon={Icon && <Icon />}>{content}</NotificationBadge>;
-};
+  component: NotificationBadge,
 
-Default.story = {
   parameters: {
     info: "Check Orbit.Kiwi for more detailed design guidelines.",
   },
-};
 
-Default.args = {
-  content: "10",
-  icon: "null",
-};
-
-Default.argTypes = {
-  icon: {
-    options: Object.keys(Icons),
-    control: {
-      type: "select",
-    },
+  args: {
+    children: "10",
+    icon: "null",
+    type: TYPE_OPTIONS.INFO,
+    ariaLabel: "additional information for screen readers",
   },
-};
-
-export const Neutral = ({ content }) => {
-  return <NotificationBadge type={TYPE_OPTIONS.NEUTRAL}>{content}</NotificationBadge>;
-};
 
-Neutral.story = {
```

Comment:
For this Story file, there were soooo many similar stories (the differences were only in `type` prop), so I squash them into one story with `type` option.